firmware/Core/Src/littlefs/littlefs_helper.c

```diff
@@ -57,22 +55,22 @@ struct lfs_file_config LFS_file_cfg = {
 
 // -----------------------------LITTLEFS FUNCTIONS-----------------------------
 
-
 /**
  * @brief Formats Memory Module so it can successfully mount LittleFS
  * @param None
  * @retval 0 on success, negative LFS error codes on failure
  */
 int8_t LFS_format()
 {
-	int8_t result = lfs_format(&LFS_filesystem, &LFS_cfg);
-	if (result < 0)
-	{
-		DEBUG_uart_print_str("Error formatting!\n");
-		return result;
+	int8_t format_result = lfs_format(&LFS_filesystem, &LFS_cfg);
+	if (format_result < 0) {
+        LOG_message(LOG_SYSTEM_LFS, LOG_SEVERITY_CRITICAL, LOG_all_sinks_except(LOG_SINK_FILE), "Error formatting FLASH memory!");
+		return format_result;
```

Comment:
Can you please convert this file's indentation to use spaces instead of tabs. There's a few places it's mixed in this repo, but in general, we prefer spaces. Ideally, never mix in the same file.